packages/stratocacher-layer-localstorage/src/index.js

```diff
@@ -0,0 +1,32 @@
+import {Layer} from "stratocacher";
+import localStorage from "store";
+
+import Q from "q";
+
+export default class LocalStorage extends Layer {
+
+	static reset() {
+		super.reset();
+		localStorage.forEach((key) => {
+			localStorage.remove(key);
+		});
+	}
+
+	get() {
+		let val = localStorage.get(this.key);
+		if (this.opt.copy) {
+			val = JSON.parse(val);
+		}
```

Comment:
Hmmm... Isn't `copy` is actually implicit with `localStorage`?  Value put into `localStorage` are automatically serialized?